composer.json

```diff
@@ -7,7 +7,7 @@
 	],
 	"require": {
 		"php": "^7.1 || ^8.0",
-		"phpstan/phpstan": "^0.12.40"
+		"phpstan/phpstan": "^0.12.49"
```

Comment:
This is the minimum version we need in order for phpstan to detect the correct type

Reply:
Yeah, it's possible that's where `count($x) >= 1` was implemented.